connect/Env.hx

```diff
@@ -114,13 +116,35 @@ class Env extends Base {
         @param config The configuration of the logger.
     **/
     public static function initLogger(config: LoggerConfig): Void {
-        if (logger == null) {
-            logger = new Logger(config);
-        } else {
-            throw "Logger instance is already initialized.";
-        }
+
+        logger.set(rootLogger,new Logger(config));
```

Comment:
It is a design requirement of the SDK that the logger cannot be reinitialized, so the part that throws the exception must be kept. In this case, we check that logger does not contain a root logger (sing its exists() method) and create it or throw an exception depending on the case.